src/test/java/org/unicode/cldr/surveydriver/SurveyDriver.java

```diff
@@ -61,10 +60,11 @@ public class SurveyDriver {
      * Enable/disable specific tests using these booleans
```

Comment:
would it make sense for these to be in a properties or json file?

Reply:
yes, there's already a surveydriver.properties which so far only has the password in it